### ISSUE TITLE
object/distributed_virtual_portgroup.go: panic on missing DVS on DVPortgroup

### ISSUE THREAD

**Issue by vancluever** [vmware/govmomi#1098]
terraform-providers/terraform-provider-vsphere#449 describes an edge case where a DVPortgroup lacks a DVS. Not too sure how it got into this state, but ultimately this is resulting in a panic at https://github.com/vmware/govmomi/blob/3f145230c6a5d20d6d09366c239b429303b1dc7e/object/distributed_virtual_portgroup.go#L47.

Just making a note here, I'm guessing short of making a case for this this in the simulator it would be pretty tough to repro and get a fix in for.

**Comment by dougm** [comment]
Seen in #1035 due to permissions, could be the same issue here?

**Comment by vancluever** [comment]
@dougm yeah possibly! I will pass a message on to the reporter. At the very least we could probably add a nil check - what do you think?

**Comment by dougm** [comment]
Yes, a nil check at least.  I did want to try reproducing to see why this field isn't included in the `MissingSet`, which would turn into an error instead of panic: https://github.com/vmware/govmomi/blob/d3ae3004773719bcc413ff8e3396f2ce6bf0827b/vim25/mo/retrieve.go#L56

**Comment by vancluever** [comment]
Sounds good! I've notified the reporter for now,
